schemas/+behavior/TowersBlockTrialVideo.m

```diff
@@ -0,0 +1,15 @@
+%{
+-> behavior.TowersBlockTrial
+---
+video_path:                 varchar(255)         # video directory + filename for each trial
```

Comment:
Is it true that each trial has a video, just to confirm

Reply:
That's true for "selected" sessions.
The idea is that the researchers generate the videos for every trial in some sessions that are of particular interest.

More on this on the next comment